components/AddWorkout/AddWorkout.js

```diff
@@ -36,35 +54,202 @@ export default function AddWorkout({ setVisible }) {
+                                    handleDeleteSelector(selector)
+                                }}><FontAwesomeIcon icon={faClose} /></DeleteButton>
+                            </ExerciseHeader>
+
+
+                            {selectedExercises[selector] && (
+                                <ExerciseSettings>
+                                    <label htmlFor={`${selector}-sets`}>Sets</label>
+                                    <Input type='number' name={`${selector}-sets`} required />
+
+                                    <label htmlFor={`${selector}-reps`}>Reps</label>
+                                    <Input type='number' name={`${selector}-reps`} required />
+
+                                    <label htmlFor={`${selector}-weight`}>Weight</label>
+                                    <Input type='number' name={`${selector}-weight`} required />
+                                </ExerciseSettings>
+                            )}
+                        </Selector>
+
+                    ))}
+                    <AddButton type="button" onClick={addSelector}>
+                        <FontAwesomeIcon icon={faPlus} />
+                        <span>Add Exercise</span>
+                    </AddButton>
+                </Form>
             </form>
-            <button onClick={() => setVisible(false)}>Cancel</button>
-        </>
+        </PopUp>
     );
-}
\ No newline at end of file
+}
+
+const ExerciseSettings = styled.div`
+font-family: verdana;
+display: flex;
+gap: 20px;
+flex-direction: column;
+margin-top: 20px;
+font-size: 14px;`
+
+const WorkoutsHeader = styled.div`
+display: flex;
+justify-content: space-between;
+margin-bottom: 20px;
+position: sticky;
+top: 0;
+padding: 20px;
+background-color: #0C0B10;; 
+`
+
+const ExerciseHeader = styled.div`
+display: flex;
+align-items: center;
+justify-content: space-between;
+`
+
+const CloseButton = styled.button`
+background-color: #292830;
+color: white;
+display: flex;
+align-items: center;
+justify-content: center;
+width: 40px;
+height: 40px;
+border-radius: 50px;
+border: none;
+font-weight: 200;
+font-size: 1rem;
+`
+const AddButton = styled.button`
+display: flex;
+color:rgb(193, 193, 193);
+gap: 10px;
+font-size: 1rem;
+align-items: center;
+justify-content: center;
+flex-direction: row;
+text-decoration: none;
+background-color: #292830;
+border-radius: 25px;
+border: none;
+padding: 10px
+`
+
+const SubmitButton = styled.button`
+background-color: #292830;
+color: white;
+display: flex;
+align-items: center;
+justify-content: center;
+width: 40px;
+height: 40px;
+border-radius: 50px;
+border: none;
+font-weight: 200;
+font-size: 1rem;
+`
+
+const DeleteButton = styled.button`
+
+background-color:rgb(53, 52, 59);
+color: white;
+display: flex;
+align-items: center;
+justify-content: center;
+width: 40px;
+height: 40px;
+border-radius: 50px;
+border: none;
+font-weight: 200;
+font-size: 1rem;
+`
```

Comment:
It looks like the only difference between these components and also the `CloseButton` is the background color.

Instead of having two separate components, I suggest creating a single styled component that adjusts the background color based on a prop. This will keep your code more efficient and maintainable while reducing duplication.